java/src/org/openqa/selenium/WebElement.java

```diff
@@ -166,6 +166,7 @@ public interface WebElement extends SearchContext, TakesScreenshot {
    * @param name The name of the attribute.
    * @return The attribute/property's current value or null if the value is not set.
    */
+  @Deprecated
   @Nullable String getAttribute(String name);
```

Comment:
Maybe an explicit warning `to use WebElement.getDomAttribute1 and `WebElemenr.getDomProperty instead` would be a good idea

Reply:
The `@deprecated` tag in the Javadoc will be helpful